cmd/print.go

```diff
@@ -219,6 +254,22 @@ func (pr *PrintRunner) initializeNamespaceFilter() error {
 	return nil
 }
 
+// 1. When --output-file receives no argument, initializeOutputFile will do nothing.
+// 2. When --output-file receives an argument, initializeOutputFile will validate whether the value is correct.
+func (pr *PrintRunner) initializeOutputFile() {
+	if pr.outputFile == "" {
+		return
+	}
```

Comment:
We need to validate that outputFile is not "" when the flag is enabled upfront. If the user provides such a flag without a value, we just exit with an error